src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.DataRecorder;
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.FileWriter;
+import core.basesyntax.service.impl.DataParserImpl;
+import core.basesyntax.service.impl.DataRecorderImpl;
+import core.basesyntax.service.impl.FileReaderImpl;
+import core.basesyntax.service.impl.FileWriterImpl;
+import core.basesyntax.strategy.Strategy;
+import core.basesyntax.strategy.impl.StrategyImpl;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.PurchaseOperation;
+import core.basesyntax.strategy.operation.QuantityOperation;
+import core.basesyntax.strategy.operation.ReturnOperation;
+import core.basesyntax.strategy.operation.SupplyOperation;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FILE_PATH_TO = "src/main/java/resources/input.csv";
+    private static final String FILE_PATH_FROM = "src/main/java/resources/recorded.csv";
```

Comment:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/2e85bab1-7aa7-4f32-92f2-1762d04f993f)

Reply:
I have created a new folder called 'resources' with the path 'src/main' and have changed the path to cvs files.